dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM nginx:latest
+
+RUN apt-get update && apt-get upgrade -y
+
+
+COPY . /usr/share/nginx/html
+
+
+EXPOSE 8080
```

Comment:
The default port for Nginx is 80, so it's recommended to expose that instead of 8080.

Reply:
My bad,  will update it for sure. Thanks.